dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -271,5 +311,20 @@ private void ConfigureAzureOpenAI(IKernel kernel)
         kernel.Config.SetDefaultTextCompletionService(azureOpenAIConfiguration.ServiceId);
     }
 
+    private void ConfigureAzureOpenAIChatAsText(IKernel kernel)
+    {
+        var azureOpenAIConfiguration = this._configuration.GetSection("AzureOpenAI").Get<AzureOpenAIConfiguration>();
+
+        Assert.NotNull(azureOpenAIConfiguration);
+
+        kernel.Config.AddAzureChatCompletionService(
+            deploymentName: azureOpenAIConfiguration.ChatDeploymentName!,
```

Comment:
are there validation checks somewhere making sure the value is not null or empty? that's a common issue with tests, not having the right configuration and figuring out what is missing

Reply:
Good point. 
Added validation for it.